subworkflows/nf-core/fastq_fastqc_umitools_fastp/tests/main.nf.test.snap

```diff
@@ -65,15 +301,71 @@
                     },
                     "unspecified"
                 ]
+            ],
+            [
+                [
+                    {
+                        "id": "test",
+                        "single_end": false
+                    },
+                    75
+                ]
+            ],
+            [
+                "versions.yml:md5,85bd0117e5778fff18e3920972a296ad",
+                "versions.yml:md5,c50aa59475ab901bc6f9a2cf7b1a14e0",
+                "versions.yml:md5,f3dcaae948e8eed92b4a5557b4c6668e"
             ]
         ],
         "meta": {
             "nf-test": "0.8.4",
             "nextflow": "23.10.1"
         },
-        "timestamp": "2024-01-12T08:38:50.08674429"
+        "timestamp": "2024-03-18T12:10:18.546963"
     },
-    "reads": {
+    "skip_trimming": {
+        "content": [
+            [
+                [
+                    {
+                        "id": "test",
+                        "single_end": false
+                    },
+                    [
+                        "/nf-core/test-datasets/modules/data/genomics/sarscov2/illumina/fastq/test_1.fastq.gz",
```

Comment:
Not sure how this ended up the snapshot, or which output it is (since the individual output snapshots aren't named), but maybe we need to fix in modules?